web/index.html

```diff
@@ -29,7 +29,8 @@
             <div class="row">
                 <div class="col-md-6">
                     <h2>Cascading Docker Image Security Updates</h2>
-                    <p>Describe me. Maybe add an image?</p>
+                    <p>How do you rebuild your docker application containers when there are dependency updates?</p>
+                    <p><img>img/Dockerfile Image Update Demo.png</img></p>
```

Comment:
This should be `<img src="img/Dockerfile Image Update Demo.png" alt="Dockerfile Image Update Demo"/>`

Reply:
no spaces in filenames plz 😄